src/interaction/navigationhandler.cpp

```diff
@@ -52,6 +52,14 @@ namespace {
     constexpr const char* KeyReferenceFrame = "ReferenceFrame";
     const double Epsilon = 1E-7;
 
+    constexpr const openspace::properties::Property::PropertyInfo OriginInfo = {
```

Comment:
I think this accidentally slipped in? The `origin` was split into `anchor` and `aim` and moved into `OrbitalNavigator`. I think all this `origin` related stuff can be removed...  (?)

Reply:
Fixed now